source/examples/rapids-sagemaker-higgs/notebook.ipynb

```diff
@@ -26,8 +26,10 @@
   },
```

Comment:
<p><code>--repository-name</code> requires an argument in this command, and none was being given. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/381/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>